apps/api/src/routes/UserRoutes.ts

```diff
@@ -19,14 +18,25 @@ userRoutes.get('/', async (req, res, next) => {
     }
 });
 
+userRoutes.get('/:id', async (req, res, next) => {
+    try {
+        const id = parseInt(req.params.id);
+
+        const userService = new UserService(database);
+        const result: User | undefined = await userService.getById(id);
```

Comment:
Instead of being able to return and undefined user, i think instead an error should be thrown in the User Service class